Longest_Increasing_Subsequence/LIS.go

```diff
@@ -0,0 +1,62 @@
+package main
```

Comment:
Please add a brief problem description at the start of file as multi line comment. Rest of the code looks good..